Sources/PT.PM.PythonParseTreeUst/PythonAntlrConverter.cs

```diff
@@ -788,19 +803,22 @@ public Ust VisitAtom(PythonParser.AtomContext context)
         {
             if (context.ChildCount == 3)
             {
-                if (context.GetChild(0).GetText() == "(")
-                {
-                    var visited = Visit(context.GetChild(1));
-                    return new ArrayCreationExpression
-                    {
-                        Initializers = visited is MultichildExpression multichild
-                            ? UstUtils.ExtractMultiChild(multichild)
-                            : new List<Expression> { visited.ToExpressionIfRequired() },
-                        TextSpan = context.GetTextSpan()
-                    };
-                }
+                var visited = Visit(context.GetChild(1));
 
-                return Visit(context.GetChild(1));
+                return new ArrayCreationExpression
+                {
+                    Initializers = visited is MultichildExpression multichild
+                        ? UstUtils.ExtractMultiChild(multichild)
+                        : new List<Expression> { visited.ToExpressionIfRequired() },
```

Comment:
To me `ToExpressionIfRequired` is a bit strange name. You require it here to be an expression, what other ifs there might be? Perhaps `ToExpressionIfNeeded` or even `AsExpression` would be a better name?

The same goes for `ToStatementIfRequired`

Reply:
It makes sense.